frontend/packages/console-shared/src/components/dashboard/dashboard-card/card.scss

```diff
@@ -8,6 +8,13 @@ $co-dashboard-text-font-size: 0.875rem;
   margin: 0;
 }
 
+.co-dashboard-card--resource-metrics .query-browser__wrapper {
```

Comment:
Should this be `co-dashboard-card__resource-metrics`? It feels like an element rather than a modifier.

http://getbem.com/naming/

It would be better if we can add a specific class name to the wrapper element here. (I'm not sure if we are able to pass a `className` through to the nested component).